templates/web/lincolnshire/report/new/roads_message.html

```diff
@@ -0,0 +1,6 @@
+    <div id="js-lincs-parish-grass" class="hidden js-responsibility-message js-roads-bucks js-roads-centralbeds js-roads-shropshire">
```

Comment:
I don't think the bucks/centralbeds/shropshire parts are needed here, they're to too with showing/hiding messages for those cobrands.